WordPress/Sniffs/VIP/SuppressFiltersSniff.php

```diff
@@ -8,61 +8,203 @@
  */
 
 /**
- * Suppress Filters need to be 'true' when getting posts using get_posts.
+ * Support_filters must be false when using get_posts(), wp_get_recent_posts(), get_children().
```

Comment:
Well, you've switched from `Suppress Filters need to be 'true'...' to `Support_filters must be false...`. Even if the change in boolean is intentional, the first word is not accurate as far as I can tell.